gammapy/estimators/map/core.py

```diff
@@ -1093,6 +1105,27 @@ def read(cls, filename, checksum=False):
         ) as hdulist:
             return cls.from_hdulist(hdulist, checksum=checksum)
 
+    def copy(self, reference_model=None):
+        """Deep copy
+
+        Parameters
+        ----------
+
+        reference_model : `~gammapy.modeling.models.SkyModel`, optional
+            The reference model to use for conversions. If None, the copied model is used.
```

Comment:
```suggestion
            The reference model to use for conversions. If None, the original model is used.
```

Reply:
I would add a word of caution here. Something like : "Flux maps have been obtained for a specific reference model. Changing it will change the fluxes. Handle with care."

Reply:
added